src/components/FormValidator.js

```diff
@@ -62,12 +73,11 @@ class FormValidator {
     this._inputList = Array.from(
       this._form.querySelectorAll(this._inputElementSelector)
     );
+
     // Find the submit button in the current form
     this._buttonElement = this._form.querySelector(this._submitButtonSelector);
     this._inputList.forEach((inputElement) => {
-      inputElement.addEventListener("input", () => {
-        this.toggleInputError(inputElement);
-      });
+      inputElement.addEventListener("change", () => this.toggleInputError());
```

Comment:
`change`


You need to use event `input` to have **live** validation during typing the text in the inputs